src/Polly.Specs/Registry/ReadOnlyPolicyRegistrySpecs.cs

```diff
@@ -282,7 +282,7 @@ public void Policies_Should_Be_Added_To_The_Registry_When_Using_Collection_Initi
                 {key, policy}
             };
 
-            testRegistry.Should().Equal(new KeyValuePair<string, IsPolicy>(key, policy));
+            testRegistry.Should().Equals(new KeyValuePair<string, IsPolicy>(key, policy));
```

Comment:
Great. I wasn't sure about this change. The compiler didn't like `Equal`.  Let try something else.

Reply:
It cannot implicit convert KVP to `IEnumerable<string, IsPolcy>` because `testRegistry` uses a dictionary. How do I run the failing test? Otherwise, should this change be ok `new[] { new KeyValuePair<string, IsPolicy>(key, policy) }`

Reply:
I guess you need to check it is equal to an equivalent dictionary/collection.

Alternatively you could assert it has a count of 1 and contains that key and value?

Reply:
Fair enough